packages/cozy-harvest-lib/src/components/TriggerManager.jsx

```diff
@@ -147,10 +181,9 @@ export class DumbTriggerManager extends Component {
       const cipherId = this.getSelectedCipherId()
       await flow.handleFormSubmit({
         client,
-        account,
+        account: account || {},
```

Comment:
if `flow.handleFormSubmit` need account to not be undefined, shouldn't it have a default parameter instead of forcing its caller to take care of passing it an empty object?